pyoffers/models/core.py

```diff
@@ -59,7 +59,7 @@ def update(self, **kwargs):
 
     @generic_method
     def delete(self, **kwargs):
-        return self._manager.delete(self.id, status='deleted')
+        return self._manager.update(self.id, status='deleted')
```

Comment:
It should be done in a separate PR, since it is a bug fix . Change log entry should be in Fixed section

Reply:
This bug is not present in master, seems like I've put it here myself in this PR

Reply:
yet, I don't see it in a diff of commit where the line has changed

Reply:
Then it should be reverted